backend/db/config.py

```diff
@@ -90,6 +91,11 @@ def get_pg_connect_url(local=False):
     # 'csets_to_ignore',
     'cset_members_items_plus',
 ]
+# STANDALONE_TABLES: Not derived from any other table, nor used to derive any other table/view. Used for QC testing.
+STANDALONE_TABLES = [
```

Comment:
## Add `STANDALONE_TABLES`
This helps us keep track of what should not be in `DERIVED_TABLE_DEPENDENCY_MAP` and is used by `TestBackendDbConfig`